pkg/dtest/manifests.go

```diff
@@ -0,0 +1,43 @@
+package dtest
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/datawire/teleproxy/pkg/k8s"
+	"github.com/datawire/teleproxy/pkg/kubeapply"
+)
+
+const msg = `
+kubeconfig does not exist: %s
+
+  Run "make claim" to acquire a kubernaut cluster, or use
+  TELEPROXY_DEV_KUBECONFIG to point elsewhere
+
+`
+
+func Manifests(kubeconfig string, files ...string) {
+	override := os.Getenv("TELEPROXY_DEV_KUBECONFIG")
+	if override != "" {
+		kubeconfig = override
+	}
+
+	if _, err := os.Stat(kubeconfig); os.IsNotExist(err) {
+		fmt.Printf(msg, kubeconfig)
+		os.Exit(1)
+	}
+
+	err := kubeapply.Kubeapply(k8s.NewKubeInfo(kubeconfig, "", ""), 30*time.Second, false, files...)
+	if err != nil {
+		fmt.Println()
+		fmt.Println(err)
```

Comment:
Does this get output as soon as it is run, or is it buffered?  Did you take any action regarding parallel tests?

Reply:
It's the least buffered you can get. It will not be buffered by the test binary itself, but it will be buffered by `go test` if there is more than one test binary running.

Reply:
I haven't taken any particular action with regard to parallel tests other than to try and have timeouts be on the shorter side.